app/assets/stylesheets/application.css

```diff
@@ -31,3 +31,12 @@
   display: flex;
   gap: 20px;
 }
+

```

Comment:
- [ ] This app functions great! However, the design aspect needs much improvements, which apparently has no styling applied to it. Please take note that it's one of the project's objective that our design closely mimics the given [template design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=). Let's improve upon the design aspect by introducing the following tweaks such as _Clean layout/ positioning, background colors, colors, typography(font family and font styling)_ to the following screens: 
  * Splash screen
  * Login and Signup screens
  * Category page
  * Transaction page, and similar pages

**Your design:**
**Your splash screen:**        |  **Your Login:**        |  **Your categories page:**     
:-------------------------:|:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/227751215-12354c7c-c266-4b38-9c39-d6f3cb96b940.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/227751257-e9038f19-fd86-445a-8428-b18e6ca229fb.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/227751305-0185b0da-f9f7-4b3a-bf98-a060bf2904f9.png">


**Expected design:**
**Original splash screen:**        |  **Original login screen:**        |  **The original transaction:**     
:-------------------------:|:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/198862103-fe9fe8a8-7a06-4a03-92f6-8528c3d4d2f1.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/227751557-e1341b09-89ea-42aa-ad37-cd38d2510b53.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/227751375-40282971-5bde-49ca-9612-8cc1be58ba99.png">